eth2-staker-checklist.md

```diff
@@ -0,0 +1,84 @@
+# Launchpad ENV variables
+
+- `ETH1_NETWORK_NAME`
+- `ETH2_NETWORK_NAME`
+
+----
+
+# Eth2 Staker Checklist
+
+## Hardware
+
+### Recommended
+
+- In order to process incoming validator deposits from the eth1 chain, you'll need to run an eth1 client in parallel to your eth2 client. While it is possible to use a third-party service like Infura, we recommend running your own client in order to ensure the network stays as decentralised as possible.
+    - For example, you need at least 140 GB SSD to run geth fast sync on *mainnet*.
+
+## Keystore
+1. - [ ] I have securely generated and saved my keystore(s).
+1. - [ ] I have backed up my mnemonic.
+1. - [ ] I have set a strong password for my keystore(s)
+1. - [ ] I have backed up my password. 
+
+## Security
+1. - [ ] I have secured the root account.
+2. - [ ] I have hardened ssh on a random port
+3. - [ ] I have set up a firewall.
+4. - [ ] I have forwarded the necessary ports from my router to the correct machine(s). Only open the ports that apply to your installation (default ports listed):
+        - [ ] Port 30303 TCP/UDP for geth
+        - [ ] Port 19000 TCP/UDP for Nimbus
+        - [ ] Port 9001 TCP/UDP for Teku
```

Comment:
```suggestion
        - [ ] Port 9000 TCP/UDP for Nimbus
        - [ ] Port 9000 TCP/UDP for Teku
```

@superphiz 
I think  19000 and 9001 are the recommended setting in Somer Esat's great guides. But they are actually 9000 by default in Nimbus and Teku.